src/main/scala/scalismo/registration/MutualInformationMetric.scala

```diff
@@ -208,7 +208,7 @@ case class MutualInformationMetric[D: NDSpace, A: Scalar](fixedImage: Field[D, A
       val derivsAtValidPoints = derivsForPoints.collect { case Some(v) => v }
       val deriv = derivsAtValidPoints.foldLeft(zeroVec)((acc, v) => acc + v)
 
-      val scaledDeriv = (1.0 / (derivsAtValidPoints.size * binSizeMovingImage)) * deriv
+      val scaledDeriv = deriv * (1.0 / (derivsAtValidPoints.size * binSizeMovingImage))
```

Comment:
1.0 /  is no longer needed. should we keep this for clarity?

Reply:
Not sure that I understand. It should still be there, but we can only put it right of the vector. For some reason with scala 3.2 the multiplication with the double on the lhs of the * operator is causing a compile time error. This is why I had to make this change everywehre.